json-resume/src/Resume.js

```diff
@@ -0,0 +1,25 @@
+import React, {Fragment} from 'react';
+import TextareaAutosize from '@material-ui/core/TextareaAutosize';
+import Input from '@material-ui/core/Input';
+import './App.css';
+import { useHistory } from "react-router-dom";
+
+
+export default function DataEntry(props) {
+    let history = useHistory();
+    let resume = props.location.state.resume;
```

Comment:
could you use localStorage for this instead of history? Not exactly sure of pros/cons but just an idea!